client/my-sites/people/followers-list/followers.jsx

```diff
@@ -110,15 +112,23 @@ class Followers extends Component {
 			);
 		}
 
-		let emptyTitle;
 		if ( this.siteHasNoFollowers() ) {
 			if ( 'email' === this.props.type ) {
-				emptyTitle = preventWidows(
-					this.props.translate( 'No one is following you by email yet.' )
+				return (
+					<Card>
+						<EmailVerificationGate
+							noticeText={ this.props.translate(
+								'You must verify your email to add subscribers.'
+							) }
+							noticeStatus="is-info"
+						>
+							<AddSubscriberForm />
+						</EmailVerificationGate>
+					</Card>
 				);
-			} else {
-				emptyTitle = preventWidows( this.props.translate( 'No WordPress.com followers yet.' ) );
 			}
+
+			const emptyTitle = preventWidows( this.props.translate( 'No WordPress.com followers yet.' ) );
```

Comment:
I know the terminology is a bit scattered, but should this be "subscribers"?

Reply:
Done ✔️